src/apps/lwaftr/lwaftr.lua

```diff
@@ -137,6 +137,14 @@ local function get_icmp_payload(ptr)
    return ptr + constants.icmp_base_size
 end
 
+-- This function converts between IPv4-as-host-uint32 and IPv4 as
+-- uint8_t[4].  It's a stopgap measure; really the rest of the code
+-- should be converted to use IPv4-as-host-uint32.
+local function convert_ipv4(addr)
+   local str = require('lib.yang.util').ipv4_ntop(addr)
```

Comment:
This is a gnarly function that I want to make go away, and I'd rather not add the following lib.protocol.ipv4 import to the global scope.  So I agree it's nasty but hopefully making it stick out will make us want to refactor to remove it :)